pkg/cmd/api/api_test.go

```diff
@@ -425,6 +449,38 @@ func Test_apiRun_paginationGraphQL(t *testing.T) {
 	assert.Equal(t, "PAGE1_END", endCursor)
 }
 
+func Test_apiRun_silent(t *testing.T) {
```

Comment:
Instead of adding a separate test function, could you add the `Silent: true` mode as another test case in `Test_apiRun` please?

Reply:
Oh. Right. I will do that!